go/test/endtoend/vreplication/resharding_workflows_v2_test.go

```diff
@@ -605,6 +605,14 @@ func testMoveTablesV2Workflow(t *testing.T) {
 
 	testVSchemaForSequenceAfterMoveTables(t)
 
+	// Confirm that the auto_increment clause on customer.cid was removed.
+	cs, err := vtgateConn.ExecuteFetch("show create table customer", 1, false)
+	require.NoError(t, err)
+	require.Len(t, cs.Rows, 1)
+	require.Len(t, cs.Rows[0], 2) // Table and "Create Table"
+	csddl := strings.ToLower(cs.Rows[0][1].ToString())
+	require.False(t, strings.Contains(csddl, "auto_increment"), "customer table still has auto_increment clause: %s", csddl)
```

Comment:
Use `require.NotContains(...)` instead of `require.False( ... strings.Contains(...) )`